vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -342,6 +342,23 @@ testCloudInit() {
   echo "$test:Finish"
 }
 
+testSourcesList() {
+  test="testSourcesList"
+  sourcesListPath="/etc/apt/sources.list"
+  searchWord="snapshot"
+  # This file should always exist on any Ubuntu node, however, still check its existence for sanity
+  if test -f "${sourcesListPath}"; then
```

Comment:
should we err if this file doesn't exist?

Reply:
I guess the test wants to ensure that the file doesn't content any snapshot references., so if there file doesnt' exist it should be fine.  

If we believe this file should always be present I would add another test which verify for this explicitly.